pkg/sql/schema_changer.go

```diff
@@ -2326,6 +2326,10 @@ type GCJobTestingKnobs struct {
 
 	// If true, the GC job will not wait for MVCC GC.
 	SkipWaitingForMVCCGC bool
+
+	// If true, disables the usage of DelRange (i.e. falls back to the legacy
+	// way).
+	DisableDelRange bool
```

Comment:
Could add a TODO (to me) to remove this when we figure out how to control the mvcc gc queue? Thanks.